src/main/java/com/rackspacecloud/metrics/ingestionservice/influxdb/LocalUUID.java

```diff
@@ -0,0 +1,9 @@
+package com.rackspacecloud.metrics.ingestionservice.influxdb;
+
+import java.util.UUID;
+
+public class LocalUUID {
+    public UUID generateUUID() {
```

Comment:
First thing, can we move this method into a class where we are using it? Also, I am not sure why do we need a method for another static method call `UUID.randomUUID()`. Second thing, if we really need this method, we can make this method `generateUUID()` a static one too.